docs/workload/elasticsearch/elasticsearch.md

```diff
@@ -0,0 +1,573 @@
+```
+
+#### Setting cStor Storage Class as default
+
+Change the default storage class from your current setting to OpenEBS cStor. For example, in this tutorial default storage class is used as `cstorsc-gqzxe` from standard.
+```bash
+kubectl patch storageclass standard -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+kubectl patch storageclass cstorsc-gqzxe -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+```
+
+#### Verify default Storage Class
+ 
+List the storage classes and verify `cstorsc-gqzxe` is set to `default`.
+```bash
+kubectl get sc
+```
+```bash
+#sample output
+NAME                        PROVISIONER                                                RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+cstorsc-gqzxe (default)     cstor.csi.openebs.io                                       Delete          Immediate              true                   139m
+openebs-device              openebs.io/local                                           Delete          WaitForFirstConsumer   false                  148m
+openebs-hostpath            openebs.io/local                                           Delete          WaitForFirstConsumer   false                  148m
+openebs-jiva-default        openebs.io/provisioner-iscsi                               Delete          Immediate              false                  148m
+openebs-snapshot-promoter   volumesnapshot.external-storage.k8s.io/snapshot-promoter   Delete          Immediate              false                  148m
+premium-rwo                 pd.csi.storage.gke.io                                      Delete          WaitForFirstConsumer   true                   158m
+standard                    kubernetes.io/gce-pd                                       Delete          Immediate              true                   158m
+standard-rwo                pd.csi.storage.gke.io                                      Delete          WaitForFirstConsumer   true                   158m
+```
+
+### Installing and Accessing Elasticsearch
+
+Set instance and namespace variables:
+```bash
+export instance_name=elastic
+export namespace_name=default
+kubectl-kudo install elastic --namespace=$namespace_name --instance $instance_name
+```
+```bash
+#sample output
+operator default/elastic created
+operatorversion default/elastic-7.0.0-0.2.1 created
+instance default/elastic created
+```
+#### Verifying Elastic pods
+
+```bash
+kubectl get pods -n $namespace_name
+```
+```bash
+#sample output
+NAME                    READY   STATUS    RESTARTS   AGE
+elastic-coordinator-0   1/1     Running   0          31s
+elastic-data-0          1/1     Running   0          56s
+elastic-data-1          1/1     Running   0          44s
+elastic-master-0        1/1     Running   0          2m31s
+elastic-master-1        1/1     Running   0          119s
+elastic-master-2        1/1     Running   0          90s
+```
+
```

Comment:
Can you please add PVC output as well if you have..

Reply:
Added pvc description